eng/GetSpecsList.ps1

```diff
@@ -0,0 +1,7 @@
+# Navigate to your local specs repo Git clone and run this script from there
+$repoPath = [regex]::Escape((Get-Location).Path)
+$partialPaths = Get-ChildItem -Path 'specification' -File -Filter 'readme.md' -Recurse |
+    Where-Object { $_.Name -clike 'readme.md' } |
+    ForEach-Object { $_.FullName -replace "^$repoPath", '' }
+$filePath = Join-Path $PSScriptRoot 'SpecsList.txt'
+$partialPaths | Out-File -FilePath $filePath
```

Comment:
This script doesn't seem to update the SmokeTestInputs.txt, does it have to be unified manually?

Reply:
Yes. It simply gets the relative path to the readmes. Currently, it isn't worth the effort to make something more complicated since our ignore strategy with the `#` stuff is there... You'd have to, for every line you wanted to add, see if the file already exists in the SmokeTestInputs and only add new lines; not to mention if you wanted them in the correct (order) position in the file. Next time we decide to update the list, hopefully there is less stuff ignored, and we can just blow out the list entirely, and re-add the ignored files by hand. At that point, it would be worth making a single-script solution.